rs/protobuf/def/state/canister_state_bits/v1/canister_state_bits.proto

```diff
@@ -312,12 +312,28 @@ message WasmChunkStoreMetadata {
   uint64 size = 2;
 }
 
+// TODO(EXC-1670): Migrate to pb_canister_state_bits::LogVisibilityV2.
+// The current enum only supports i32 values, which limits the
+// storage of allowed_viewers principals. LogVisibilityV2 will
+// support both enum values and a list of principals.
 enum LogVisibility {
   LOG_VISIBILITY_UNSPECIFIED = 0;
   LOG_VISIBILITY_CONTROLLERS = 1;
   LOG_VISIBILITY_PUBLIC = 2;
 }
 
+message LogVisibilityAllowedViewers {
+  repeated types.v1.PrincipalId principals = 1;
+}
+
+message LogVisibilityV2 {
+  oneof log_visibility_v2 {
+    int32 controllers = 1;
```

Comment:
I agree that setting those tags looks excessive especially because it does not seem to be used anywhere. But this is still the closest option semantically to the Rust version so I would prefer to keep it this way.